examples/reading_dense_layouts.py

```diff
@@ -68,31 +63,58 @@ def write_array():
     ctx = tiledb.Ctx()
     # Open the array and write to it.
     with tiledb.DenseArray(ctx, array_name, mode='w') as A:
-        data = np.array(([1, 2, 3, 4],
-                         [5, 6, 7, 8],
-                         [9, 10, 11, 12],
-                         [13, 14, 15, 16]))
+        # NOTE: global writes are not currently supported in the Python API.
+        # The following code will produce the same array as the corresponding
+        # C++ example in the docs (which wrote in global order)
+        data = np.array(([1, 2, 5, 6],
+                         [3, 4, 7, 8],
+                         [9, 10, 13, 14],
+                         [11, 12, 15, 16]))
         A[:] = data
 
 
-def read_array():
+def read_array(order):
     ctx = tiledb.Ctx()
 
     # Open the array and read from it.
     with tiledb.DenseArray(ctx, array_name, mode='r') as A:
         # Get non-empty domain
-        print(A.nonempty_domain())
+        print("Non-empty domain: {}".format(A.nonempty_domain()))
 
         # Slice only rows 1, 2 and cols 2, 3, 4.
-        data = A.query(attrs=["a"], coords=True)[1:3, 2:5]
+        data = A.query(attrs=["a"], order=order, coords=True)[1:3, 2:5]
```

Comment:
Worth adding a comment that the `.query()` syntax is required in order to specify coords for dense arrays and the ordering.